pkg/util/fileoperator.go

```diff
@@ -81,6 +82,38 @@ func (r RealFileOperator) Updatefile(fileName, updatedVal, searchString string,
 	return errors.Errorf("failed to find %s in file %s", searchString, fileName)
 }
 
+// UpdateOrAppendMultipleLines will update or append multiple lines based on the
+// given string
+func (r RealFileOperator) UpdateOrAppendMultipleLines(fileName string,
+	keyUpdateValue map[string]string, perm os.FileMode) error {
+	buffer, err := ioutil.ReadFile(filepath.Clean(fileName))
+	if err != nil {
+		return errors.Wrapf(err, "failed to read %s file", fileName)
+	}
+	lines := strings.Split(string(buffer), "\n")
+
+	// TODO: We can split above read file into key value pairs and later we can
+	// append with \n and update file
+	// TODO: Use regular expresion to replcae key value pairs
+	// will be doing after current blockers
+	for key, updatedValue := range keyUpdateValue {
+		found := false
```

Comment:
pin required  for `key`and `updateValue` ?

Reply:
It is not required since it is not a pointer